analyses/infercnv-consensus-cell-type/references/diagnosis-celltype-groups.tsv

```diff
@@ -0,0 +1,15 @@
+diagnosis_group	celltype_groups
+B-cell leukemia	dendritic cell, innate lymphoid cell, macrophage, monocyte, myeloid, natural killer cell, plasma cell, T cell, endothelial cell, pericyte
+T-cell leukemia	B cell, dendritic cell, innate lymphoid cell, macrophage, monocyte, myeloid, natural killer cell, plasma cell, endothelial cell, pericyte
+Myeloid leukemia	B cell, dendritic cell, innate lymphoid cell, macrophage, natural killer cell, plasma cell, T cell
+Mixed-type leukemia	dendritic cell, innate lymphoid cell, macrophage, natural killer cell, plasma cell, endothelial cell, pericyte

```

Comment:
A few comments here: 
- I think we should add fibroblast here. I don't know how often they'll show up in leukemias, but I also don't think endothelial cells and pericytes will be there. They are for sure not leukemia though and if they are there because they are circulating in the blood then I think we want to capture that. 
- For myeloid we should probably remove any myeloid cell types. As we know, different myleoid cells can resemble each other so I would remove macrophage and dendritic cell from that list. 
- The mixed-type is really hard... I don't feel confident in really including any myeloid or lymphoid lineage cells there which may mean we won't have enough to actually run inferCNV, but I think for those it might be okay. There's only 5 samples that fall into this group.

Reply:
> The mixed-type is really hard... I don't feel confident in really including any myeloid or lymphoid lineage cells there which may mean we won't have enough to actually run inferCNV, but I think for those it might be okay. There's only 5 samples that fall into this group.

I think this would be a fine outcome if these samples don't get run through. It's a _hard_ one...